bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestBasicOps.java

```diff
@@ -258,18 +258,19 @@ public void testMutateRowCommon() throws IOException {
     Result expectedReturn = mutateRowHasResult() ? Result.EMPTY_RESULT : null;
 
     // Test multiple Puts
+    long timestamp = 10_000;
     Result result =
         mutateRow(
             table,
             new RowMutations(testRowKey)
                 .add(
                     (Mutation)
                         new Put(testRowKey)
-                            .addColumn(COLUMN_FAMILY, "q".getBytes(), "v1".getBytes()))
+                            .addColumn(COLUMN_FAMILY, "q".getBytes(), timestamp, "v1".getBytes()))
```

Comment:
Why is a timestamp needed now?

Reply:
to allow delete to work later in the test